sql/updates/world/3.3.5/2016_82_44_05_world.sql

```diff
@@ -0,0 +1,92 @@
+(13,7,30657,0,0,31,0,3,17474,0,0,0,0,'','Effect_0, Effect_1, Effect_2 hits Target Trigger');
+
+DELETE FROM `spell_linked_spell` WHERE `spell_trigger`=30658;
+INSERT INTO `spell_linked_spell` (`spell_trigger`, `spell_effect`, `type`, `comment`) VALUES
+(30658, 30571, 0, 'Quake Trigger Quake Knockback');
+
+-- Making a summon group for Magtheridon
+DELETE FROM `creature` WHERE `id`=17256;
+DELETE FROM `creature_summon_groups` WHERE `summonerId`=17257; -- Magtheridon
+INSERT INTO `creature_summon_groups` (`summonerId`,`summonerType`,`groupId`,`entry`,`position_x`,`position_y`,`position_z`,`orientation`,`summonType`,`summonTime`) VALUES
+(17257,0,1,17256,-31.7645,-35.8374,0.714268,1.37881 ,6,6000), -- Hellfire Channelers
+(17257,0,1,17256,10.3369,-19.7942,0.714241,2.53073  ,6,6000),
+(17257,0,1,17256,10.4777,23.9914,0.714224,3.66519 ,6,6000),
+(17257,0,1,17256,-31.9479,40.2201,0.714251,4.93928  ,6,6000),
+(17257,0,1,17256,-55.6891,2.17024,0.71428,0.05236,6,6000);
+
+DELETE FROM `spell_script_names` WHERE `ScriptName` IN('spell_magtheridon_blaze_target','spell_magtheridon_shadow_grasp','spell_magtheridon_shadow_grasp_visual');
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+(30541,'spell_magtheridon_blaze_target'),
+(30410,'spell_magtheridon_shadow_grasp'),
+(30166,'spell_magtheridon_shadow_grasp_visual');
+
+DELETE FROM `spell_linked_spell` WHERE `spell_trigger`=-30410 AND `spell_effect`=44032 AND `type`=0;
+
+UPDATE `creature_template` SET `flags_extra`=128, `ScriptName`='npc_magtheridon_room' WHERE `entry`=17516;
+
+UPDATE `spell_dbc` SET `AttributesEx3`=262400, `EffectImplicitTargetA1`=53, `EffectImplicitTargetB1`=0, `EffectRadiusIndex1`=16, `MaxAffectedTargets`=1,  `RangeIndex`=13 WHERE `Id`=30630;
+
+DELETE FROM `creature_text` WHERE `entry`=17257;
+INSERT INTO `creature_text` (`entry`,`groupid`,`id`,`text`,`type`,`language`,`probability`,`emote`,`duration`,`sound`,`BroadcastTextId`,`TextRange`,`comment`) VALUES
+(17257,0,0,'Wretched, meddling insects! Release me, and perhaps I will grant you a merciful death!',14,0,100,0,0,10247,17339,0,'Magtheridon - Taunt01'),
+(17257,1,0,'Vermin! Leeches! Take my blood and choke on it!',14,0,100,0,0,10248,17340,0,'Magtheridon - Taunt02'),
+(17257,2,0,'Illidan is an arrogant fool! I will crush him and reclaim Outland as my own!',14,0,100,0,0,10249,17341,0,'Magtheridon - Taunt03'),
+(17257,3,0,'Away, you mindless parasites! My blood is my own!',14,0,100,0,0,10250,17342,0,'Magtheridon - Taunt04'),
+(17257,4,0,'How long do you believe your pathetic sorcery can hold me?',14,0,100,0,0,10251,17343,0,'Magtheridon - Taunt05'),
+(17257,5,0,'My blood will be the end of you!',14,0,100,0,0,10252,17344,0,'Magtheridon - Taunt06'),
```

Comment:
hmm, i saw about blood furnace in wowwiki.
how much random this text are? any suggestion about timer?